projects/cm_mcu/LocalTasks.c

```diff
@@ -656,3 +656,230 @@ void init_registers_ff()
   apollo_i2c_ctl_reg_w(3, 0x21, 1, 0x03, 1, 0x01); //  00000001 [P17..P10]
 }
 #endif // REV2
+
+#ifdef REV2
+int init_load_clk(int clk_n)
+{
+
+  if (getPowerControlState() != POWER_ON) {
+	  log_debug(LOG_SERVICE, "LGA80D_init() needs more time to turn on power supplies \r\n");
+	  vTaskDelay(pdMS_TO_TICKS(3000)); // wait 3s
+  }
+  clk_n = clk_n -1;
+  int PreambleList_row[5] = {6,0,3,0,0};
+  int RegisterList_row[5] = {378,0,587,0,0};
+  int PostambleList_row[5] = {3,0,5,0,0};
```

Comment:
please a) add a comment what these numbers are supposed to be and b) assuming I understand what they are, I thought they were going to be stored in the EEPROM?

Reply:
Okay, this makes sense. I will make MCU be independent of EEPROM by storing the triplet of these numbers which are the number of registers in each list in the last EEPROM page of each clock. 

So, there won't be hardcoded like this. Instead, three registers will be read out from EEPROM and converted to decimal numbers.